packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1083,14 +1444,18 @@ export class TransactionController extends BaseControllerV1<
    *
    * @param transaction - The transaction params to estimate gas for.
    * @param multiplier - The multiplier to use for the gas buffer.
+   * @param networkClientId - The network client id to use for the estimate.
    */
   async estimateGasBuffered(
+    // NOTE(JL): Need to update SwapsController's usage of this method
```

Comment:
drop this comment. note it elsewhere like in a ticket

Reply:
working on this

Reply:
https://github.com/MetaMask/MetaMask-planning/issues/1019

Reply:
https://github.com/MetaMask/core/pull/3643/commits/ea2fbe1266bbeb01f6299fa8273f44f870c51c89